### PR TITLE
Ruler: add option to skip writing result to ingesters

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does

We have a testing use case where we need to run ruler without writing results to ingesters (or ingest storage). In this PR I propose to add a config option to allow to disable it.

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [ ] Tests updated.
- [ ] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

**Review by dimitarvdimitrov — APPROVED** [review]

File: docs/sources/mimir/configure/configuration-parameters/index.md [hunk 1]
```diff
@@ -2118,6 +2118,11 @@ tenant_federation:
 # group runtime duration must exceed 50.0% of the evaluation interval.
 # CLI flag: -ruler.independent-rule-evaluation-concurrency-min-duration-percentage
 [independent_rule_evaluation_concurrency_min_duration_percentage: <float> | default = 50]
+
+# (experimental) False to disable writing the result of rule evaluation to
```
[the remark this answers is not in the document]
Comment (pracucci) [reply docs/sources/mimir/configure/configuration-parameters/index.md:2122]: Exactly, setting to false will disable...

File: docs/sources/mimir/configure/configuration-parameters/index.md [hunk 2]
```diff
@@ -2118,6 +2118,11 @@ tenant_federation:
...
+# ingesters. This option can be used for testing purposes.
```
[the remark this answers is not in the document]
Comment (pracucci) [reply docs/sources/mimir/configure/configuration-parameters/index.md]: Done in https://github.com/grafana/mimir/pull/9060/commits/6d956e50586c11bf16a8015cf9ca6d2a19660693

Comment (tacole02) [inline docs/sources/mimir/configure/configuration-parameters/index.md]:
```suggestion
# (experimental) Writes the results of rule evaluation to
# ingesters. Use this option for testing purposes. To disable, set to `false`.
```

**Review by tacole02 — APPROVED** [review]

File: docs/sources/mimir/configure/configuration-parameters/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tacole02) [reply docs/sources/mimir/configure/configuration-parameters/index.md:2122]: Thanks for clarifying! I suggested a small update.

File: docs/sources/mimir/configure/configuration-parameters/index.md (hunk 2 above)
Comment (pracucci) [reply docs/sources/mimir/configure/configuration-parameters/index.md]: Done in https://github.com/grafana/mimir/pull/9060/commits/407ec7da36aa06b3cd9ed0627bb40daa9cb82e1a

**Pull request merged**